test/specs/create-local-vue.spec.js

```diff
@@ -232,3 +233,32 @@ describeWithShallowAndMount('createLocalVue', mountingMethod => {
     }
   )
 })
+
+// TODO: Clean this up, it's only useful while debugging the issue.
+describe.only('Test', () => {
+  // When setting the array length value to `1`, then the issue disappears.
+  // However, when setting it to anything bigger, the issue appears.
+  const sets = Array.from({ length: 2 })
+
+  it.each(sets)(
+    // Note, the message may not be the final one, based on what I discover.
```

Comment:
can you put link to the issue? it will be difficult to remember what this test is really doing